core/controllers/reader.py

```diff
@@ -421,6 +421,75 @@ class AnswerSubmittedEventHandler(base.BaseHandler):
     """Tracks a learner submitting an answer."""
 
     REQUIRE_PAYLOAD_CSRF_CHECK = False
+    URL_PATH_ARGS_SCHEMAS = {
+        'exploration_id': {
+            'schema': {
+                'type': 'basestring',
+                'validators': [{
+                    'id': 'is_regex_matched',
+                    'regex_pattern': constants.ENTITY_ID_REGEX
+                }]
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'params': {
+                'schema': {
+                    'type': 'object_dict',
+                    'validation_method': (
+                        domain_objects_validator.validate_params_dict),
+                },
+                'default_value': {}
+            },
+            'session_id': {
+                'schema': {
+                    'type': 'basestring'
+                }
+            },
+            'old_state_name': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'has_length_at_most',
+                        'max_value': constants.MAX_STATE_NAME_LENGTH
+                    }]
+                }
+            },
+            'answer': {
+                'schema': {
+                    'type': 'basestring'
+                }
+            },
+            'client_time_spent_in_secs': {
+                'schema': {
+                    'type': 'float',
+                    'validators': [{
+                        'id': 'is_at_least',
+                        'min_value': 0
+                    }]
+                }
+            },
+            'answer_group_index': {
+                'schema': {
+                    'type': 'int'
+                }
+            },
+            'rule_spec_index': {
+                'schema': {
+                    'type': 'int'
+                }
+            },
+            'version': {
+                'schema': editor.SCHEMA_FOR_VERSION
+            },
+            'classification_categorization': {
+                'schema': {
+                    'type': 'basestring'
+                }
+            }
```

Comment:
Isn't there some list of categories that we can check here?

Reply:
Okay, there are only 4 classifications, right?

Reply:
you can use `choices` option, see other schema to see how is it used

Reply:
Okay, thank you. Please have a look now.